client/modules/IDE/actions/ide.js

```diff
@@ -269,14 +269,6 @@ export function startSketch() {
   };
 }
 
-export function startAccessibleSketch() {
-  return (dispatch) => {
-    dispatch(clearConsole());
-    dispatch(startAccessibleOutput());
-    dispatch(startSketchAndRefresh());
-  };
-}
-
```

Comment:
I need to look into the full chain of actions which result from calling `startAccessibleSketch()` and see which are them are and are not necessary.  The `startAccessibleOutput()` is the one which can be removed but idk about `clearConsole()` and `startSketchAndRefresh()` and how this compares to calling the normal `startSketch()`.

It might be that we modify this function instead of removing it.